docs/release.rst

```diff
@@ -0,0 +1,38 @@
+Restraint Release Notes
```

Comment:
Rename to something like "Prepare Release Guide"

Reply:
or just Release Guide I guess